src/metabase/types.clj

```diff
@@ -127,6 +127,7 @@
 (derive :type/State :type/Category)
 (derive :type/Country :type/Category)
 (derive :type/Name :type/Category)
+(derive :type/Title :type/Category)
```

Comment:
how is a title a category?

Reply:
We have name as a category, seemed similar enough that I followed for consistency reasons.

Reply:
Name should definitely not be a category.

Reply:
k. I'll fix it in the original automagic dashboards PR as it's there that I added title, and changed a bunch of other stuff wrt types.